### PR TITLE
Reset password UI

### PR DESCRIPTION
**Pull request by washibana**
- French localization

### CODE REVIEW

File: backend/Origam.Server/Resources/SharedResources.cs-CZ.resx [hunk 1]
```diff
@@ -417,4 +417,7 @@
   <data name="TwoFactorCodeRequired" xml:space="preserve">
     <value>Kód pro dvoufaktorové přihlášení musí být vyplněn</value>
   </data>
+  <data name="Reset" xml:space="preserve">
+    <value>Obnovit</value>
```
Comment (tvavrda) [inline backend/Origam.Server/Resources/SharedResources.cs-CZ.resx:421]: Where is this used? I would expect the text to be a bit longer. Something like "Reset your password" or "Did you forget your password?".

**Review by tvavrda — CHANGES_REQUESTED** [review]

File: backend/Origam.Server/Resources/SharedResources.cs-CZ.resx (hunk 1 above)
Comment (tvavrda) [reply backend/Origam.Server/Resources/SharedResources.cs-CZ.resx:421]: Then I suggest to make the text longer, as is usual today on the websites. I think the most proper is "Did you forget your password?".

Comment (washibana) [reply backend/Origam.Server/Resources/SharedResources.cs-CZ.resx:421]: Screenshot 2022-03-02 at 11 46 49

**Comment by tvavrda** [comment]
At least Czech translation should spell "E-mail" instead of "Email" which means "enamel".

**Comment by washibana** [comment]
> At least Czech translation should spell "E-mail" instead of "Email" which means "enamel".

https://github.com/origam/origam/blob/da742fb27028036397bb424a40873f894bf21c68/backend/Origam.Server/Resources/SharedResources.cs-CZ.resx#L285-L287

**Review by tvavrda — APPROVED** [review]

**Pull request merged**